src/Components/test/E2ETest/ServerRenderingTests/FormHandlingTests/FormWithParentBindingContextTest.cs

```diff
@@ -1466,6 +1466,8 @@ public void SubmitButtonFormenctypeAttributeOverridesEnhancedFormEnctype()
     [Fact]
     public void EnhancedFormThatCallsNavigationManagerRefreshDoesNotPushHistoryEntry()
     {
+        GoTo("about:blank");
+
```

Comment:
I noticed that with my new E2E test, even before I implemented the fix, it would falsely pass on retry if I didn't explicitly force the browser to start on a different URL. That's because the browser instance is reused across retries.

So to show the same false-passing doesn't happen here either, I've also made it explicitly start on a different URL.